src/Apps/Order/Routes/Offer/index.tsx

```diff
@@ -192,7 +180,18 @@ export class OfferRoute extends Component<OfferProps, OfferState> {
                 }
                 Sidebar={
                   <Flex flexDirection="column">
-                    <TransactionSummary order={order} mb={[2, 3]} />
+                    <TransactionSummary
+                      order={order}
+                      mb={[2, 3]}
+                      offerOverride={
+                        this.state.offerValue &&
+                        this.state.offerValue.toLocaleString("en-US", {
+                          style: "currency",
+                          currency: "USD",
+                          minimumFractionDigits: 0,
```

Comment:
There's another ticket in Jira for adding '.00' after the price here, so you could change `minimumFractionDigits` to 2 here and take care of that already.